src/js/components/reviewData/ReviewDataNotifyModal.jsx

```diff
@@ -107,6 +111,23 @@ export default class ReviewDataNotifyModal extends React.Component {
             });
     }
 
+    openClientEmail(e) {
+        e.preventDefault();
+        const toEmails = (this.state.selectedUsers.map((user) => user.email)).join(', ');
+        const subject = 'DATA Act Broker - Submission Ready for Review';
+        const revUser = this.props.fromUser.toUpperCase();
+        const revAgecny = this.props.submittingAgency.toUpperCase();
+        const revLink = window.location.href;
+        console.log(revLink);
+        const body = [
+            `${revUser} has shared a DATA Act broker submission with you from ${revAgecny}. `,
+            `Follow this link (${revLink}) to review their submission. `,
+            `For questions or comments, please visit the Service Desk at https://servicedesk.usaspending.gov/ `,
+            `or e-mail DATAPMO@fiscal.treasury.gov.`
+        ].join('');
+        window.location.href = `mailto:${toEmails}?subject=${subject}&body=${body}`;
```

Comment:
Prefer `history.push` instead of directly setting `window.location.href`

Reply:
Just wondering why we're setting it here at all instead of just making it a mailto link?

Reply:
Yeah I thought it'd be easier to switch functions as solutions for the future if/when we roll it back. Considering it'd be looping in the history var from the highest parents, I'm going to try Alisa's direct mailto link.